src/App.tsx

```diff
@@ -29,20 +29,21 @@ const App = () => {
 
   let routes;
 
-  if (authenticated !== undefined)
+  if (authenticated !== undefined) {
     if (authenticated) {
       routes = <AuthenticatedRoute />;
     } else {
       routes = <UnauthenticatedRoute />;
     }
```

Comment:
Very minor thing. Do you think something like this is valid?
```suggestion
  if (authenticated !== undefined && authenticated) {
    routes = <AuthenticatedRoute />;
  } else {
      routes = <UnauthenticatedRoute />;
  }
```

Reply:
@rathorevaibhav Actually we don't want to assign any routes, hence the current logic. In the above scenario, we will set UnAuthenticated routes.

Reply:
Sounds good then.